packages/go-kosu/.goreleaser.yml

```diff
@@ -26,5 +26,15 @@ archives:
   - id: kosu
     name_template: "{{ .ProjectName }}-{{ .Os }}"
 
+dockers:
+  - dockerfile: Dockerfile.binaries
+    goos: linux
+    goarch: amd64
+    binaries:
+      - kosud
+      - kosu-cli
+    image_templates:
+      - gcr.io/kosu-test-network/go-kosu:{{ .ShortCommit }}
```

Comment:
should we add `go-kosu:latest` tag as well?

Reply:
good point